mdsuite/calculators/einstein_helfand_ionic_conductivity.py

```diff
@@ -161,8 +164,7 @@ def _apply_operation(self, ensemble, index):
         -------
         MSD of the tensor_values.
         """
-        msd = (ensemble - (
-            tf.repeat(tf.expand_dims(ensemble[0], 0), self.data_range, axis=0))) ** 2
+        msd = tf.math.squared_difference(ensemble, ensemble[None, 0])
```

Comment:
Why not use `Calculator._msd_operation`

Reply:
Yes, the dimension expansion is along differing axis so the option is either to have the computation in the calculator itself, or to have quite a large logic segment in the main _msd_operation method. We would need logic for square=True/False and then a check if system=True logic section under each. It isn't bad but it didn't seem elegant. I can think of a way to integrate it all.

Reply:
Okay, in that case I don't see a reason, why it should be changed. Looks all good